runtime/stdlib/builtin.go

```diff
@@ -138,3 +140,97 @@ var LogFunction = NewStandardLibraryFunction(
 var HelperFunctions = StandardLibraryFunctions{
 	LogFunction,
 }
+
+var CreatePublicKeyFunction = NewStandardLibraryFunction(
+	sema.PublicKeyTypeName,
+	&sema.FunctionType{
+		Parameters: []*sema.Parameter{
+			{
+				Label:          sema.PublicKeyPublicKeyField,
+				Identifier:     sema.PublicKeyPublicKeyField,
+				TypeAnnotation: sema.NewTypeAnnotation(&sema.VariableSizedType{Type: &sema.UInt8Type{}}),
+			},
+			{
+				Label:          sema.PublicKeySignAlgoField,
+				Identifier:     sema.PublicKeySignAlgoField,
+				TypeAnnotation: sema.NewTypeAnnotation(sema.SignatureAlgorithmType),
+			},
+		},
+		ReturnTypeAnnotation:  sema.NewTypeAnnotation(sema.PublicKeyType),
+		RequiredArgumentCount: sema.RequiredArgumentCount(2),
```

Comment:
This function shouldn't have any optional (variable) arguments, right? If so, this line should be removed

Reply:
Yeah, it shouldn't. I thought its a dynamic validation for any parameter to avoid inconsistencies. Will remove it!